### PR TITLE
python3.pkgs.qiling: init at 1.2.3

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120006` at 81511c97 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.python-registry</li>
    <li>python38Packages.qiling</li>
    <li>python39Packages.python-registry</li>
    <li>python39Packages.qiling</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/python-registry/default.nix:22:0:
```
   |
22 |     description = "Pure Python parser for Windows Registry hives";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/qiling/default.nix:36:0:
```
   |
36 |     description = "Qiling Advanced Binary Emulation Framework";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/qiling/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    python-registry
+    keystone-engine
+    pyelftools
+    gevent
+  ];
+
+  # Tests are broken (attempt to import a file that tells you not to import it,
+  # amongst other things)
+  doCheck = false;
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/qiling/default.nix:34]: Missing pythonImportsCheck.

File: pkgs/development/python-modules/qiling/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+, fetchPypi
+, capstone
+, unicorn
+, pefile
+, python-registry
+, keystone-engine
+, pyelftools
+, gevent
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/qiling/default.nix:12]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/python-registry/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, enum-compat
+, unicodecsv
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-registry/default.nix:7]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/python-registry/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "99185f67d5601be3e7843e55902d5769aea1740869b0882f34ff1bd4b43b1eb2";
+  };
+
+  propagatedBuildInputs = [
+    enum-compat
+    unicodecsv
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-registry/default.nix:20]: Missing checkPhase and pythonImportsCheck.

**Comment by cole-h** [comment]
qiling is broken on aarch64 because capstone does not support it (it creates a manylinux2014-x86_64 wheel).

**Comment by SuperSandro2000** [comment]
> qiling is broken on aarch64 because capstone does not support it (it creates a manylinux2014-x86_64 wheel).

Can you mark capstone broken on aarch64?

**Comment by cole-h** [comment]
Sure can!

**Pull request merged**